staging/src/k8s.io/cri-api/pkg/apis/testing/fake_runtime_service.go

```diff
@@ -396,7 +394,7 @@ func (r *FakeRuntimeService) StartContainer(containerID string) error {
 
 	c, ok := r.Containers[containerID]
 	if !ok {
-		return fmt.Errorf("container %s not found", containerID)
+		return fmt.Errorf("No such container: %s", containerID)
```

Comment:
The fact that these errors are being changed and no test changes are required suggests to me that we're missing sufficient test coverage to be confident in this change.